File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/incoder-1/minimum_spanning_tree.java
--- 
+++ 
@@ -19,7 +19,7 @@
             minSpanningTree.add(edge);
             groupByNode = update(groupByNode, vertex_u, vertex_v);
             for (Node node : groupByNode.get(vertex_v)) {
-                groupByNode = update(groupByNode, node, vertex_u);
+                groupByNode.put(node,groupByNode.get(vertex_u));
             }
         }
     }

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/codex-sh/minimum_spanning_tree.py
--- 
+++ 
@@ -8,6 +8,6 @@
             mst_edges.add(edge)
             group_by_node[u].update(group_by_node[v])
             for node in group_by_node[v]:
-                group_by_node[node].update(group_by_node[u])
+                group_by_node[node] = group_by_node[u]
 
     return mst_edges
